Sources/Components/ARCListItem.swift

```diff
@@ -26,28 +33,39 @@ public struct ARCListItem<Leading: View, Trailing: View>: View {
         labelTitle: String? = nil,
         @ViewBuilder leading: @escaping () -> Leading,
         @ViewBuilder trailing: @escaping () -> Trailing,
-        onTap: @escaping () -> Void
+        onTap: @escaping () -> Void,
+        background: Color? = Color.arcWhite,
+        titleForeground: Color? = Color.arcBlack,
+        subtitleForeground: Color? = Color.arcDarkGray,
+        borderColor: Color? = Color.arcLightGray,
+        verticalPadding: CGFloat? = nil
```

Comment:
Just checking we don't want to do the same as the other properties and default to `.ArcListItem.verticalPadding`

Reply:
@BenShutt So I wanted it to have a default `.ArcListItem.verticalPadding` however it errored out as it's an internal property and it doesn't let to be used as default value.

I've set it to null here temporarily but it does have a default property in ui
`.padding(.vertical, verticalPadding ?? .ArcListItem.verticalPadding)`

Reply:
We can make it `public` I think? After that I think it's good to roll